patchwork-registries/src/main/java/net/minecraftforge/registries/ForgeRegistryEntry.java

```diff
@@ -31,6 +31,14 @@ public final IForgeRegistryEntry setRegistryName(Identifier name) {
 		return this;
 	}
 
+	public final IForgeRegistryEntry setRegistryName(String name) {
+		return setRegistryName(new Identifier(name));
+	}
+
+	public final IForgeRegistryEntry setRegistryName(String modID, String name) {
+		return setRegistryName(modID + ":" + name);
```

Comment:
This should just construct an `Identifier` directly, instead of combining the strings together only for them to be split again in the constructor of `Identifier`.